CHANGELOG.md

```diff
@@ -0,0 +1,1104 @@
+    * Visuals: Rocket Fuel
+    * Visuals: Displaying Braking Status
+    * Vehicle: Toyota - Enforce Stock Longitudinal Control
+    * Subaru: Increase Steering Torque
+    * Longitudinal: Acceleration Personality
+    * UI: Display CPU Temperature on Sidebar
+    * Lateral: Block Lane Change with Road Edge Detection
+    * UI: Display DM Camera in Reverse Gear
+    * UI: Auto-hide Selected UI Elements
+    * Visuals: Display End-to-End Longitudinal Status
+    * Toyota: Stop and Go Hack (alpha)
+    * Visuals: Onroad Settings
+    * Honda: Serial Steering Support
+    * Volkswagen: Non-ACC Platforms Support
+    * Longitudinal: Dynamic Personality
+    * Honda Nidec: Allow Stock Longitudinal Control
+    * Lateral Planner: Dynamic Lane Profile
+    * Lateral: Custom Camera and Path Offsets
+    * Toyota: Door Controls
+* Major Features & Systems
+  * Modular Assistive Driving System (MADS)
+    * Complete driving assistance framework
+  * Driving Model Manager
+    * Custom driving model selection with support for about 86 models (as of writing), from Optimus Prime (September 2023) up to The Cool People’s Models (October 2025)
+  * Neural Network Lateral Control (NNLC)
+    * Advanced torque-based lateral control
+  * Dynamic Experimental Control (DEC)
+    * Intelligent longitudinal control adaptation
+  * Speed Limit Assist (SLA)
+    * Comprehensive speed limit integration featuring `mapd` for offline map limits downloads, a Speed Limit Resolver for sourcing data (from car, map, combined, etc), on-screen UI for Speed Limit Information/Warning, and Speed Limit Assist (SLA) to adjust cruise speed automatically.
+  * Intelligent Cruise Button Management (ICBM)
+    * System designed to manage the vehicle’s speed by sending cruise control button commands to the car’s ECU.
+  * Smart Cruise Control Map & Vision (SCC-M / SCC-V)
+    * When using any form of long control (openpilot longitudinal or **ICBM**) it will control the speed at which you enter and perform a turn by leveraging map data (**SCC-M**) and/or by leveraging what the model sees about the curve ahead (**SCC-V**)
+  * Vehicle Selector
+    * If your vehicle isn’t fingerprinted automatically, you can still use the vehicle selector to get it working
+  * sunnylink Integration
+    * Cloud connectivity and settings backup/restore
+  * External Storage Support
+    * Expanded storage options
+  * mapd Integration
+    * Allow downloading maps for your area, which could be useful for Speed Limit Assist (SLA)
+  * ❌ REMOVED: Navigate on openpilot (NoO)
+    * Navigate on openpilot (NoO) has been removed as upstream is prioritizing improving the driving model’s capabilities and simplifying the training stack.
+    * The feature will return in a future release once model improvements from upstream make it more reliable.
```

Comment:
I wouldn't make this promise, i would probably say something like "The feature will _likely_ return..." and add like "and upstream implements it" to make it clear we don't plan to go our of our way to make it